docs/examples/dimensions.rst

```diff
@@ -558,33 +566,33 @@ dimension. This feature should however be considered experimental.
     productTable = Dimension(
```

Comment:
Should we use CachedDimensions instead?

Reply:
I do not see any downside to using a CachedDimension in this case, so I will update the example before merging it to master.